server/server.go

```diff
@@ -116,8 +127,23 @@ func StartServer() {
 
 	port := ":8295"
 	srv := http.Server{Addr: port, Handler: chi.ServerBaseContext(serverCtx, r)}
+
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, syscall.SIGTERM)
+	go func() {
+		<-sig
+		log.Info().Msg("SIGTERM received, disabling health check")
+
+		healthCheckActive = false // disable health check
+
+		time.Sleep(60 * time.Second)
+		srv.Shutdown(serverCtx)
```

Comment:
Just checking - we're disabling the health check and giving ALB enough time to stop routing traffic here before calling `Shutdown`, right?  Are we guaranteed 60s before stop instance is killed?  What about open connections?  Are we given any guarantees about those when we receive `SIGTERM`?